Jenkinsfiles/utils.groovy

```diff
@@ -166,37 +169,37 @@ def is_read_only_db() {
     }
 }
 
-def migrate_db() {
+def migrate_db(target='') {
     /*
      * Migrate the database (only for kuma and writeable databases).
      */
     if ((get_repo_name() == 'kuma') && !is_read_only_db()) {
```

Comment:
The function `get_target_script()` is also used within `is_read_only_db()`, so that function's signature needs to change, and `target` needs to be passed into its call here.

Reply:
Fixed in ab12738810e72d1d05fbc200d26f8f519484e5d4